packages/components/src/internal/components/ontology/OntologyTreePanel.tsx

```diff
@@ -60,24 +178,33 @@ export const OntologyTreePanel: FC<OntologyTreeProps> = props => {
                 }
             );
         },
-        [root]
+        [root, filters]
```

Comment:
similar to other comment, this might be worth a comment as to why `filters` is here even though it isn't used in the callback